hapi/test/test_contact_lists.py

```diff
@@ -0,0 +1,71 @@
+import unittest2
+import random
+
+from nose.plugins.attrib import attr
+
+import helper
+from hapi.contact_lists import ContactListsClient
+from hapi.contacts import ContactsClient
+from test_contacts import ContactsClientTestCase
+
+
+class ConstactListsClientTestCase(unittest2.TestCase):
+    
+    """ Unit tests for the HubSpot Contact Lists API Python client.
+
+    This file contains some unittest tests for the Contact Lists API.
+
+    Questions, comments: http://developers.hubspot.com/docs/methods/lists/create_list
+    """
+
+    test_portal_id = 62515
+
+    def setUp(self):
+    	self.client = ContactListsClient(**helper.get_options())
```

Comment:
What do we get from helper.get_options() ? I tried pip install helper, but no get_options() method found.